components/homme/src/preqx/share/prim_advance_mod.F90

```diff
@@ -825,9 +756,13 @@ subroutine advance_hypervis_dp(elem,hvcoord,hybrid,deriv,nt,nets,nete,dt2,eta_av
                  ! normalize so as to conserve IE
                  ! scale by 1/rho (normalized to be O(1))
                  ! dp/dn = O(ps0)*O(delta_eta) = O(ps0)/O(nlev)
+#if 0
                  dpdn(:,:) = ( hvcoord%hyai(k+1) - hvcoord%hyai(k) )*hvcoord%ps0 + &
                       ( hvcoord%hybi(k+1) - hvcoord%hybi(k) )*elem(ie)%state%ps_v(:,:,nt)
                  ttens(:,:,k,ie) = ttens(:,:,k,ie) * hvcoord%dp0(k)/dpdn(:,:)
+#else
+                 ttens(:,:,k,ie) = ttens(:,:,k,ie) * hvcoord%dp0(k)/elem(ie)%state%dp3d(:,:,k,nt)
+#endif
```

Comment:
@bartgol :  this code change I believe should also be made in preqx_kokkos, but I cant locate this code.  Can you point me to it?

Reply:
Ah! I believe we do not have that adjustment for `nu_p=0` in preqx_kokkos. I can add either add it within this PR or do it in a subsequent one.